packages/atlas/src/views/viewer/VideoView/CommentsSection.tsx

```diff
@@ -257,6 +266,7 @@ export const CommentsSection: React.FC<CommentsSectionProps> = ({ disabled, vide
         value={commentBody}
         withoutOutlineBox
         onChange={(e) => setCommentBody(e.currentTarget.value)}
+        onCancel={() => handleCancelConfirmation(() => setCommentBody(''))}
```

Comment:
the main comment input in the video page doesn't use `onCancel` or has a cancel button on design